test/common/api_utils.py

```diff
@@ -274,5 +285,4 @@ def get_by_string(source_dict, search_string, default_if_not_found=None):
             dict_obj = next(val for key, val in dict_obj.iteritems() if key == search_key)
         except StopIteration:
             return default_if_not_found
-    return dict_obj
-        
\ No newline at end of file
+    return dict_obj
```

Comment:
nit: missing newline (which has to be annoying, since it came from cleaning up the extra spaces on the last line! Sorry!)